src/main/java/org/example/examplefinalProject/service/ClassRoomService.java

```diff
@@ -18,31 +17,14 @@ public ClassRoomService(ClassRoomRepository classRoomRepository, StudentReposito
         this.studentRepository = studentRepository;
     }
 
-    /**
-     * Vrací seznam všech tříd.
-     *
-     * @return Seznam tříd.
-     */
     public List<ClassRoom> findAll() {
         return classRoomRepository.findAll();
     }
 
-    /**
-     * Vrací detail třídy.
-     *
-     // @param id Identifikátor třídy.
-     * @return Detail třídy nebo {@code null}.
-     */
     public ClassRoom findById(int classRoomId) {
         return classRoomRepository.findById(classRoomId).orElse(null);
     }
 
-    /**
-     * Vrací počet studentů v dané třídě.
-     *
-     // @param id Identifikátor třídy.
-     * @return Počet studentů.
-     */
```

Comment:
Why you are deleting these JavaDocs? I think they are helpful

Reply:
navráceno